ably/internal/ablyutil/strings_test.go

```diff
@@ -0,0 +1,132 @@
+package ablyutil_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/ably/ably-go/ably/internal/ablyutil"
+)
+
+func Test_string(t *testing.T) {
+	t.Run("String array Shuffle", func(t *testing.T) {
+		t.Parallel()
+
+		strList := []string{}
+		shuffledList := ablyutil.Shuffle(strList)
+		areEqual := reflect.DeepEqual(strList, shuffledList)
+		if !areEqual {
+			t.Errorf("%v should be equal to %v", strList, shuffledList)
+		}
+
+		strList = []string{"a"}
+		shuffledList = ablyutil.Shuffle(strList)
+		areEqual = reflect.DeepEqual(strList, shuffledList)
+		if !areEqual {
+			t.Errorf("%v should be equal to %v", strList, shuffledList)
+		}
+
+		strList = []string{"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l"}
+		shuffledList = ablyutil.Shuffle(strList)
+		areEqual = reflect.DeepEqual(strList, shuffledList)
+		if areEqual {
+			t.Errorf("%v is equal to %v", strList, shuffledList)
```

Comment:
How can we assert that these are not equal?

Reply:
I didn't get